app/Http/Controllers/Auth/AuthController.php

```diff
@@ -86,6 +86,49 @@ public function login(LoginRequest $loginRequest)
         ], 200);
     }
 
+    /**
+     * Login with Google
+     *
+     * @param Request $request
+     * @return Response
+     */
+    public function loginWithGoogle(Request $request)
+    {
+        $client = new \Google_Client();
+        $client->setClientId(config('constants.gapi_client_id'));
```

Comment:
I had created a google.php under /config for Google related constants. 
We could use that here.

Reply:
Okay, sure. Didn't know about that.